dot/node.go

```diff
@@ -204,14 +210,54 @@ func NewNode(cfg *Config, ks *keystore.GlobalKeystore) (*Node, error) {
 		return nil, fmt.Errorf("failed to create state service: %s", err)
 	}
 
+	gd, err := stateSrvc.Base.LoadGenesisData()
+	if err != nil {
+		return nil, err
+	}
+
+	telemetryMailer, err := setupTelemetry(cfg, gd)
```

Comment:
Do you need to setupTelemetry for both `InitNode` and `NewNode`? :thinking:

Reply:
Yes, `InitNode` calls the `state.Initialise()` function that calls `NewBlockStateFromGenesis(db, header, telemetry)` and this last function uses the `telemetry.SendMessage(NewNotifyFinalizedTM)`

Reply:
I asked @edwardmack to do some refactoring around this.  I don't see the need for both `InitNode` and `NewNode` to be called.